schema/schema.sql

```diff
@@ -78,7 +78,12 @@ CREATE TABLE factfinder_metadata (
 DROP TABLE IF EXISTS decennial_dictionary;
 CREATE TABLE decennial_dictionary (
   variablename text, 
-  producttype text,
+ const { CUR_YEAR, PREV_YEAR } = require('../special-calculations/data/constants');
+
+function stringifyIds(ids) {
+  if (Array.isArray(ids)) return `'${ids.join("','")}'`;
+  return `'${ids}'`;
+} producttype text,
```

Comment:
Actually, I'm a little confused - is this SQL or JavaScript? Not sure how this works

Reply:
this is javascript -- pretty much exactly your stringify function, but i added handling for if `ids` was one value instead of an array, because I'm using the same query for single and aggregate profile selections. 

I think that it might be better tho to format the whole where clause for geoids in a function, because I believe SQL engines can perform much better given `column = VALUE` as opposed to `column IN (VALUE)` , and why not help the SQL engine out?

Reply:
@julialucyhogan I guess I'm still confused because I don't know how this would parse:
https://github.com/NYCPlanning/labs-factfinder-api/blob/7d0b125ce186ea335d0544dfb3f91a976f3bfe59/schema/schema.sql#L79-L93

It's like a random JS function was pasted into the SQL query D: sorry if I'm still missing something.